.github/workflows/gradle.yml

```diff
@@ -0,0 +1,25 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Java CI with Gradle
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Build Transformer
+      run: cd dev/transformer && ./gradlew build
```

Comment:
I would put this in a script (chmod +x) in .github/scripts/ to avoid having these details in the workflow yaml.

https://github.com/bndtools/bnd/blob/master/.github/scripts/build.sh

Reply:
Done with latest